File: Problem-06/missingNumber.java
/*
Author : Ayush Agrawal
College : BITS Pilani - KK Birla Goa Campus
Year/Department : 3rd year - Electronics and Instrumentation
E-Mail Id : f20170604@goa.bits-pilani.ac.in
*/

import java.util.*;
public class Main
{
    public static void main(String[] args) 
    {
        Scanner ob = new Scanner(System.in);
        int n = ob.nextInt();
        int a[] = new int[n];
        int ans = 1;
        for(int i=1;i<=n;i++) {
            a[i-1] = ob.nextInt();
            ans += i+1;
            ans -= a[i-1];
        }
        System.out.println(ans);
    }
}

File: Problem-06/missing_number.cpp
/*
Author : <Shashank>
College : <Birla Institute of Technology,Mesra>
Year/Department : <2nd>/<Computer Science and Engineering>
E-Mail Id : <jhashashank944@gmail.com>
*/

#include<iostream>
using namespace std;

int main() {
    int n;
    cin>>n;
    int arr[n-1];
    int sum = 0;
    for(int i = 0;i < n-1;i++) {
        cin>>arr[i];
        sum += arr[i];
    }
    int tot_sum = n*(n+1)/2;
    int num = tot_sum - sum;
    cout<<num;
    return 0;
}
